src/functionalTest/groovy/org/cadixdev/gradle/licenser/LicenserPluginFunctionalTest.groovy

```diff
@@ -29,14 +29,40 @@ import org.gradle.testkit.runner.TaskOutcome
 import org.junit.Rule
 import org.junit.rules.TemporaryFolder
 import spock.lang.Specification
+import spock.lang.Unroll
 
 import java.nio.file.Paths
 
 class LicenserPluginFunctionalTest extends Specification {
     @Rule
     TemporaryFolder temporaryFolder = new TemporaryFolder()
 
-    def "can run licenseCheck task"() {
+    static final def standardArguments = ["--warning-mode", "fail", "--stacktrace"].asImmutable()
+
+    static final def configurationCacheTestMatrix = [
+            // TODO: restore android plugin once versions that support configuration cache are available
+            /* gradleVersion | androidVersion | extraArgs */
+            [ "6.8.3",         null,            ["--configuration-cache"] ],
```

Comment:
Should we add earlier Gradle 6.x versions too?

I recently added more comprehensive backward-compatibility tests in another Gradle plugin:
https://github.com/n0mer/gradle-git-properties/blob/84da08e52ac0c7c19d95b2e1486804fb8f7ff1e7/src/test/groovy/com/gorylenko/BackwardCompatibilityFunctionalTest.groovy#L22-L38

Reply:
I decided against that -- it would increase the amount of time needed to execute tests, and the point of compatibility here is in order to make sure people can use licenser even if they have some other issue preventing them from updating Gradle, which is generally only the case between major versions. 

It's very unlikely a user using Gradle 6.4 could not trivially update to Gradle 6.8, so imo not worst the extra cost of testing (not to mention the disk space required for all the different Gradle distributions)